File: Solutions/C#/Accepted/136-single-number.cs
/*
Problem Title: 136 - Single Number
Lang: C#
Runtime: 92 ms
Memory: 26.4 MB
Problem Url: https://leetcode.com/problems/single-number
Submission Url: https://leetcode.com/problems/single-number/submissions/318162715
*/

public class Solution {
    public int SingleNumber(int[] nums) {
        int result = 0;
        
        foreach(int num in nums){
            result = result ^ num;
        }
        
        return result;
    }
}

File: Solutions/C++/Accepted/136-single-number.cpp
/*
Problem Title: 136 - Single Number
Lang: C++
Runtime: 12 ms
Memory: 7.7 MB
Problem Url: https://leetcode.com/problems/single-number
Submission Url: https://leetcode.com/problems/single-number/submissions/318785383
*/

class Solution {
public:
    int singleNumber(vector<int>& nums) {
        
        
        int result = 0;
        
        for(int i =0; i < nums.size(); i++){
            
            
            result = (result ^ nums[i]);
            
        }
        
        
        return result;
    }
};



